generators/component/index.js

```diff
@@ -138,13 +138,8 @@ module.exports = class extends Generator {
 
   _writeCustomComponent () {
     this.log('Creating files...')
-    
-    const destDir = `${this.themePath}/Components/${this.nameUpperCamelCase}/`
 
-    this.fs.copy(
-      this.templatePath(`*.jpg`),
-      destDir
-    )
+    const destDir = `${this.themePath}/Components/${this.nameUpperCamelCase}/`
 
     this.fs.copyTpl(
       this.templatePath(`!(*.jpg)`),
```

Comment:
we don't need this jpg restriction anymore either

Reply:
same goes for [this line](https://github.com/flyntwp/generator-flynt/pull/24/files#diff-7a02e891a6044e839b1b05f79f8171a4R161) below